native/package.json

```diff
@@ -73,7 +73,7 @@
     "react-native-permissions": "3.0.1",
     "react-native-progress": "^4.0.3",
     "react-native-reanimated": "^2.0.0",
-    "react-native-render-html": "^5.1.0",
+    "react-native-render-html": "6.0.0-alpha.24",
```

Comment:
As you can see it is still an alpha version. If you prefer waiting for the major release I will delay this PR. When I was testing the renderer I didn't encounter any actual problems.

Reply:
Not urgent at all, right @Taggotty? More of a refactoring issue of removing our custom workarounds.
So I think I tend to agree with @SteffiStuffel especially because of our refactoring in two weeks.

Reply:
That's fine to me. Then I will close this issue for now and keep the branch locally.